### PR TITLE
chore(experiments): named workspaces - add draft and menu icon

### PR DESCRIPTION
**Pull request by jo-fang**
https://wandb.atlassian.net/browse/WB-16378

### CODE REVIEW

**Review by jwlee64 — APPROVED** [review]

File: weave-js/src/assets/icons/icon-menu.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M4.25 6C4.25 5.58579 4.58579 5.25 5 5.25H19C19.4142 5.25 19.75 5.58579 19.75 6C19.75 6.41421 19.4142 6.75 19 6.75H5C4.58579 6.75 4.25 6.41421 4.25 6Z" fill="#2B3038"/>
```
Comment (jamie-rasmussen) [inline weave-js/src/assets/icons/icon-menu.svg:2]: The fills here should be using `currentColor` - the addicon script can fix this for you.

**Review by jamie-rasmussen — CHANGES_REQUESTED** [review]

File: weave-js/src/assets/icons/icon-menu.svg (hunk 1 above)
Comment (jo-fang) [reply weave-js/src/assets/icons/icon-menu.svg]: Yes thank you. I finally git cloned `addicon` and saw that script tries to change XLarge to Xlarge (fyi)

Screenshot 2023-11-14 at 5 36 26 PM

**Review by jamie-rasmussen — APPROVED** [review]

File: weave-js/src/assets/icons/icon-menu.svg (hunk 1 above)
Comment (jo-fang) [reply weave-js/src/assets/icons/icon-menu.svg]: @jamie-rasmussen do you think there's a way to enforce using addicon when someone submits a PR?

Comment (jamie-rasmussen) [reply weave-js/src/assets/icons/icon-menu.svg]: I haven't wanted to try and force people to setup and use a script that I just unofficially made. Maybe if I reduce the setup burden by cleaning it up more and moving it into the devtools repo. I've just been opting for advertising its existence, but if we wanted to do more we could either use CODEOWNERS to force review on these files by people who will insist on it or with more effort convert the checks it does to lint rules.

**Pull request merged**